app/controllers/billing_controller.rb

```diff
@@ -0,0 +1,45 @@
+class BillingController < ApplicationController
+  def index; end
+
+  def new
+    respond_to :js
+  end
+
+  def create
+    respond_to do |format|
+      if current_user.stripe_id.nil?
+        customer = Stripe::Customer.create(email: current_user.email)
+        current_user.update_columns stripe_id: customer.id
+      end
+
+      card_token = params[:stripeToken]
+      format.html {redirect_to billing_path, error: t(".fails")} if card_token.nil?
+
+      customer = Stripe::Customer.new current_user.stripe_id
+      customer.source = card_token
+      if customer.save
+        flash[:success] = t "success"
+      else
+        flash[:danger] = t "fails"
+      end
+      format.html {redirect_to success_path}
+    end
+  end
+	
+  def success; end
+
+  def payment
+    customer = Stripe::Customer.new current_user.stripe_id
+    hiring = current_user.hirings.last
```

Comment:
sao lại lấy .last e nhỉ

Reply:
Câu truy vấn này lấy ra hóa đơn mà người dùng vừa mới tạo để thanh toán ạ

Reply:
@gdhaison nếu last mà ko tồn tại thì sao em

Reply:
@hungct-1121 Về cơ bản là nó sẽ luôn tồn tại, vì trước khi chạy vào hàm này thì phải tạo đc hóa đơn thành công ạ. Nhưng để chắc chắn thì e viết thêm đoạn check

Reply:
thêm validate là ok nhé